src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -37,11 +38,13 @@ public Datastore() {
   }
 
   /** Stores the Message in Datastore. */
-  public void storeMessage(Message message) {
+  public void storeMessage(Message message)
+  {
     Entity messageEntity = new Entity("Message", message.getId().toString());
     messageEntity.setProperty("user", message.getUser());
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
+    messageEntity.setProperty("recipient", message.getRecipient()); //added this. ai think it's to store recipient's messages
```

Comment:
Comment here is not necessary. Notice that the other lines don't have a comment indicating that the recipient is stored in the message entity.